wazo_calld/plugins/voicemails/api.yml

```diff
@@ -320,6 +320,25 @@ paths:
         '503':
           $ref: '#/responses/AnotherServiceUnavailable'
   /users/me/voicemails/greetings/{greeting}:
+    post:
+      summary: Create a custom greeting
+      description: '**Required ACL:** `calld.users.me.voicemails.greetings.{greeting}.create`'
+      parameters:
+      - $ref: '#/parameters/VoicemailGreeting'
+      consumes:
+      - audio/wav
+      tags:
+      - users
+      - voicemails
+      responses:
+        '204':
+          description: The greeting was created successfully
+        '400':
+          $ref: '#/responses/InvalidRequest'
+        '404':
+          $ref: '#/responses/NotFoundError'
+        '503':
+          $ref: '#/responses/AnotherServiceUnavailable'
```

Comment:
documentation missing for `/voicemails/{voicemail_id}/greetings/{greeting}`

Reply:
done